Runtime/Scripts/Skillprint.cs

```diff
@@ -42,10 +42,16 @@ public static Skillprint Instance
         }
     }
 
-    public void Init(string GameId)
+    public void Init(string GameId, string playerId)
```

Comment:
C# supports optional parameters, can't this just be 1 signature with `string playerId=""`? also please be consistent in style for parameter capitalization; i think most C# style guides suggest camelCase parameter names so i think in this case what you added is right and the `GameId` should be `gameId`